### ISSUE TITLE
Duda Manejo de inputs 

### ISSUE THREAD

**Issue by mariaignaciag** [RodrigoToroIcarte/IIC2113-2023-1#12]
Hola Profesor, una pregunta, podemos asumir que se podrá un input válido en los test? es decir, entre los casos validos de mazos y que finalmente se presionará 4 para rendirse?

**Comment by RodrigoToroIcarte** [comment]
Sí, los test cases secretos son muy parecidos a los test cases que vienen con el código base. Si pasas los test cases del código base sin "hacer trampa" deberías pasar los test cases secretos también. 

En todos los test cases si ambos mazos son válidos lo primero que hace el jugador es rendirse. También siempre se elige un mazo que aparezca en la lista. Dicho eso, el código que les di de la vista chequea que el input del usuario sea válido y, de no serlo, le pide que ingrese otro input :nerd_face:
